tempesta_fw/sched/tfw_sched_hash.c

```diff
@@ -433,6 +433,15 @@ tfw_sched_hash_del_srv(TfwServer *srv)
 		call_rcu_bh(&cl->rcu, tfw_sched_hash_put_srv_data);
 }
 
+void
+tfw_sched_hash_refcnt(bool get)
+{
+	if (get)
+		try_module_get(THIS_MODULE);
```

Comment:
This is very generic code, may be better to have it in `main.c`?

Reply:
@ikoveshnikov  why do we need to get modules? I supposed that there is no way to unload a scheduler module if tempesta_fw.ko is loaded and references them...

Reply:
`tempesta_fw` module is referenced by schedulers modules, not vice versa:
```
root@debian:/home/user/tempesta/tempesta# ./scripts/tempesta.sh --start
Starting Tempesta...
...enable RPS on lo
...enable RPS on enp3s0
...load Tempesta modules
Loading Tempesta kernel modules...
Loading module tempesta_tls 
Loading module tempesta_db 
Loading module tempesta_fw tfw_cfg_path=/home/user/tempesta/tempesta/etc/tempesta_fw.conf
Loading module tfw_sched_hash 
Loading module tfw_sched_http 
Loading module tfw_sched_ratio 
...start Tempesta FW
done
root@debian:/home/user/tempesta/tempesta# lsmod 
Module                  Size  Used by
tfw_sched_ratio        20480  0
tfw_sched_http         16384  0
tfw_sched_hash         16384  0
tempesta_fw          4734976  3 tfw_sched_ratio,tfw_sched_hash,tfw_sched_http
tempesta_db            40960  2 tempesta_fw,tfw_sched_hash
tempesta_tls          462848  1 tempesta_fw
...
root@debian:/home/user/tempesta/tempesta# rmmod tfw_sched_ratio
root@debian:/home/user/tempesta/tempesta# lsmod 
Module                  Size  Used by
tfw_sched_http         16384  0
tfw_sched_hash         16384  0
tempesta_fw          4734976  2 tfw_sched_hash,tfw_sched_http
tempesta_db            40960  2 tempesta_fw,tfw_sched_hash
tempesta_tls          462848  1 tempesta_fw
...
```
So manual module unloading is possible.